.github/workflows/checkin.yml

```diff
@@ -0,0 +1,31 @@
+name: pull-request
+on:
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+      - uses: actions/checkout@v2
+      - name: Set up node 
+        uses: actions/setup-node@v1
+      - name: Install dependencies
+        run: |
+          id
+          whoami
+          ps -ef
+          pwd
+          sudo apt-get update -qq
+          sudo apt-get -y install -qq automake autopoint xsltproc libselinux1-dev gettext expect
+          sudo apt-get -y install -qq byacc libtool
+      - name: Run tests 
+        run: |
+          cat /proc/self/uid_map
+          cat /proc/self/status
+          systemd-detect-virt
+          ./autogen.sh --without-selinux --disable-man
```

Comment:
it's better to break these distinct steps up.  should have one run that does autogen, one for make, one for make install, and one for running the tests.